### PR TITLE
Genesis creation instructions

### PR DESCRIPTION
**Pull request by paolocappelletti**
Genesis creation tutorial: how to re-generate the genesys chain spec.

### CODE REVIEW

File: docs/3-tutorials/04-genesis-creation.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+To verify its correctness, anyone can follow the instructions in this tutorial and regenerate the same data.
+
+## Prerequisites:
+
+The following software is needed:
+
+1. Fully synched ZEND Mainchain node with dump support enabled:
+
+    - Donwload latest ZEND release from here: https://github.com/HorizenOfficial/zen/releases
```
Comment (cbonaco1) [inline docs/3-tutorials/04-genesis-creation.md:15]:
```suggestion
    - Download latest ZEND release from here: https://github.com/HorizenOfficial/zen/releases
```

File: docs/3-tutorials/04-genesis-creation.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    - Follow the instructions here https://github.com/HorizenLabs/migration-tools/tree/main/snapshot to patch the ZEND node in order to have dump support enabled
```
Comment (cbonaco1) [inline docs/3-tutorials/04-genesis-creation.md:16]: not sure if this is expected, but this link shows 404 page for me.

File: docs/3-tutorials/04-genesis-creation.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    - Compile the node, start and have it fully synched with the old mainchain 
+    
+
+2.  Fully synched EON Chain node with dump support enabled:
```
Comment (cbonaco1) [inline docs/3-tutorials/04-genesis-creation.md:20]: i think this should be `synced`
```suggestion
2.  Fully synced EON Chain node with dump support enabled:
```

File: docs/3-tutorials/04-genesis-creation.md (hunk 2 above)
Comment (paolocappelletti) [reply docs/3-tutorials/04-genesis-creation.md:16]: it's because is still private.
we have to open it before announcing the migration - cc @lander86

**Pull request merged**